spec/display-buffer-spec.coffee

```diff
@@ -128,6 +128,20 @@ describe "DisplayBuffer", ->
           expect(displayBuffer.tokenizedLineForScreenRow(4).tokens[0].isSoftWrapIndentation).toBeTruthy()
           expect(displayBuffer.tokenizedLineForScreenRow(4).tokens[1].isSoftWrapIndentation).toBeTruthy()
 
+      describe "when editor.softWrapHangingIndent is set", ->
+        beforeEach ->
+          atom.config.set('editor.softWrapHangingIndent', 3)
+
+        it "further indents wrapped lines", ->
+          expect(displayBuffer.tokenizedLineForScreenRow(11).text).toBe "       sort(left).concat(pivot).concat(sort(right)"
+          expect(displayBuffer.tokenizedLineForScreenRow(12).text).toBe "       );"
+
+        it "correctly tokenizes hanging indentation spaces", ->
```

Comment:
Some suggestions for clarifying this example:

* Talk about the content of each token.
* Assert that the token at index 4 is *not* soft-wrap indentation to clarify the boundary.
* Maybe mention why we're breaking the indentation up in the description: maybe "breaks soft-wrap indentation into a token for each indentation level to support indent guides".